skimage/feature/tests/test_fisher_vector.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+
+import numpy as np
+
+from skimage.feature.fisher_vector import (
+    learn_gmm, compute_fisher_vector, FisherVectorException,
+    DescriptorException
+)
+
```

Comment:
In order to skip everything in this file when scikit-learn is not installed, you could add

`pytest.importorskip('sklearn')` at the top of this file. Then pytest will mark all of these tests as skipped instead of failures on environments where scikit-learn has not been installed.

Reply:
@grlee77 Done as per above

Reply:
Done